include/odp/api/spec/stash_types.h

```diff
@@ -156,13 +156,40 @@ typedef struct odp_stash_capability_t {
 	 */
 	uint32_t max_stashes;
 
-	/** Maximum number of object handles per stash
+	/** Maximum common number of object handles per stash for any object size
 	 *
-	 *  The value of zero means that limited only by the available
+	 *  An application is able to store at least this many objects when using any of the
+	 *  supported object sizes. The per object size values listed in 'max_num' may be larger
+	 *  than this common value. The value of zero means that limited only by the available
 	 *  memory size.
 	 */
 	uint64_t max_num_obj;
 
+	/** Maximum number of object handles per stash for each object size
+	 *
+	 *  Only values up to 'max_obj_size' are valid. The value of zero means that limited only by
```

Comment:
"Only values up to maximum supported object handle size" could be more clear than "max_obj_size"